### PR TITLE
[runner] Append verbose level feature

### PR DESCRIPTION
**Pull request by YongseopKim**
- lv0: prints the only result
- lv1: prints result and messages btw run
- lv2: prints all of messages possible

ONE-DCO-1.0-Signed-off-by: Yongseop Kim <yons.kim@samsung.com>

### CODE REVIEW

**Comment by YongseopKim** [comment]
for #2845 #2805

**Comment by YongseopKim** [comment]
PTAL @YiHyunJin

**Comment by YongseopKim** [comment]
FYI
```
dragon@loki:~/Works/github/ONE$ ./Product/out/bin/nnpackage_run ./benchmark_nnpkg_models/mobilenet_v2_1.0_224/ -w5 -r100
===================================
MODEL_LOAD   takes 0.387 ms
PREPARE      takes 14.316 ms
EXECUTE      takes 48.587 ms
- MEAN     :  48.587 ms
- MAX      :  54.144 ms
- MIN      :  47.445 ms
- GEOMEAN  :  48.572 ms
===================================

dragon@loki:~/Works/github/ONE$ ./Product/out/bin/nnpackage_run ./benchmark_nnpkg_models/mobilenet_v2_1.0_224/ -w5 -r100 -v0
===================================
MODEL_LOAD   takes 1.726 ms
PREPARE      takes 38.706 ms
EXECUTE      takes 48.425 ms
- MEAN     :  48.425 ms
- MAX      :  54.474 ms
- MIN      :  47.301 ms
- GEOMEAN  :  48.410 ms
===================================
```

```
dragon@loki:~/Works/github/ONE$ ./Product/out/bin/nnpackage_run ./benchmark_nnpkg_models/mobilenet_v2_1.0_224/ -w5 -r100 -v1
... warmup 1 takes 58.57 ms
... warmup 2 takes 52.163 ms
... warmup 3 takes 49.654 ms
... warmup 4 takes 52.322 ms
... warmup 5 takes 48.439 ms
... run 1 takes 54.673 ms
... run 2 takes 49.267 ms
... run 3 takes 49.945 ms
...
... run 99 takes 55.001 ms
... run 100 takes 49.349 ms
===================================
MODEL_LOAD   takes 1.725 ms
PREPARE      takes 36.673 ms
EXECUTE      takes 51.170 ms
- MEAN     :  51.170 ms
- MAX      :  79.435 ms
- MIN      :  48.339 ms
- GEOMEAN  :  50.984 ms
===================================
```

```
# for now, v2 is same to v1
dragon@loki:~/Works/github/ONE$ ./Product/out/bin/nnpackage_run ./benchmark_nnpkg_models/mobilenet_v2_1.0_224/ -w5 -r100 -v2
```

**Comment by YongseopKim** [comment]
ping @Samsung/nnfw_committers

File: tests/tools/nnpackage_run/src/args.h [hunk 1]
```diff
@@ -74,6 +75,7 @@ class Args
   bool _mem_poll;
   bool _write_report;
   bool _print_version = false;
+  int _verbsoe_level;
```
Comment (wateret) [inline tests/tools/nnpackage_run/src/args.h:78]:
```suggestion
  int _verbose_level;
```

Comment (YongseopKim) [reply tests/tools/nnpackage_run/src/args.h:78]: Oops! Thank you. I'll fix it soon.

**Comment by YongseopKim** [comment]
PTAL @YiHyunJin and @wateret

**Pull request merged**